tests/test_cases/expected_kdl/slashdash_newline_before_node.kdl

```diff
@@ -1 +0,0 @@
-

```

Comment:
Looks like you accidentally included an old version of this expectation; current testsuite has a newline in it.

Reply:
Ah, I somehow thought the file had two newlines (and removed the second one).

Reply:
Oooh, the file contains CR LF, not just LF. (That's why my test is failing)

That can't be on purpose, can it?